tests/integration/object/bucketowner/bucketowner.go

```diff
@@ -341,47 +339,8 @@ func TestObjectBucketClaimBucketOwner(t *testing.T, k8sh *utils.K8sHelper, insta
 
 		// the rgw admin api is used to verify bucket ownership
 		t.Run("setup rgw admin api client", func(t *testing.T) {
-			output, err := installer.Execute("radosgw-admin", []string{"user", "info", "--uid=dashboard-admin", fmt.Sprintf("--rgw-realm=%s", objectStore.Name)}, objectStore.Namespace)
```

Comment:
Does the moved code already have other unit tests covering it, or should this unit test move with the refactor?

Reply:
I see, sounds good to remove those unit tests since setup of the client is outside their scope.